packages/periphery/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@uma/periphery",
+  "version": "1.0.0",
+  "description": "UMA Core Peripheral Scripts",
+  "dependencies": {
```

Comment:
@mrice32 @chrismaree should any or all of these be `devDeps`? I'm not really sure how to think about it. Also, maybe none of these even need to be deps if they are imported by the root `package.json`

Reply:
I think it makes sense to have these as `dependencies`. if there were any tests in this package, that had their own packages to facilitate the tests, then they would be dev dependencies.

Reply:
Is this package expected to be installed via npm at all? If not, we could set it to private so it doesn't get published.

I agree with @chrismaree. If the scripts are considered "production" code for this package, then their deps should be deps, and not devdeps.

Reply:
hmm i could see it being published to npm because some of the scripts or utils might be helpful elsewhere. Are there downsides to publishing an additional package?